t/unicode-multibyte.t

```diff
@@ -0,0 +1,23 @@
+use lib '.';
+use t::Helper;
+use Mojo::File 'path';
+use Mojo::Util 'encode';
```

Comment:
Can you use `qw(...)` instead of single quotes for consistency?

Reply:
Will change